### PR TITLE
Config database and .env

### PR DESCRIPTION
**Pull request by dzhurey**
WHAT'S THIS PR DO?
- add postgres adapter to database dev and test config
- use `.env` file to store environment config

HOW SHOULD THIS BE MANUALLY TESTED?
- create your `.env` as your machine configuration and do `source .env` from your terminal and run project

### CODE REVIEW

File: assets/package-lock.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```
Comment (awcodify) [inline assets/package-lock.json]: Is this included in the scope of this PR?

File: config/dev.exs [hunk 2]
```diff
@@ -68,8 +68,9 @@ config :phoenix, :plug_init_mode, :runtime
...
-  username: "postgres",
-  password: "postgres",
-  database: "awcodify_dev",
-  hostname: "localhost",
+  username: System.get_env("DEV_DB_USERNAME"),
+  password: System.get_env("DEV_DB_PASSWORD"),
+  database: System.get_env("DEV_DB_NAME"),
+  hostname: System.get_env("DEV_DB_HOST"),
+  port: System.get_env("DEV_DB_PORT"),
   pool_size: 10
```
Comment (awcodify) [inline config/dev.exs]: add this into ENV, too

File: assets/package-lock.json (hunk 1 above)
Comment (dzhurey) [reply assets/package-lock.json]: That should be included, I will add app key to `.env` too

is there any other credential or whatever that could added to `.env`?

Comment (awcodify) [reply assets/package-lock.json]: i mean all changes in package-lock. Why we should update much of our dependencies for this PR?

Comment (dzhurey) [reply assets/package-lock.json]: I am sorry, salah baca :D

**Comment by awcodify** [comment]
actually it is not a elixir way. Are we gonna use this way or elixir way?

**Comment by awcodify** [comment]
We are going to use elixir way

**Pull request closed without merging**